src/index.html

```diff
@@ -20,14 +20,15 @@
     <meta property="og:title"         content="We Vote" />
     <meta property="og:description"   content="View your ballot. Learn from friends. Share your vision." />
     <meta property="og:image"         content="https://WeVote.US/img/global/logos/we-vote-logo-wordmark-vertical-inverted-color-256x256.png" />
-    <link href="/css/loading-screen.css" rel="stylesheet" type="text/css" />
+    <base href="/">
+    <link href="/css/loading-screen.css" type="text/css" />
 
-    <link rel="apple-touch-icon" sizes="180x180" href="/img/global/icons/apple-touch-icon.png">
```

Comment:
Hi @jarodpeachey, thank you for this pull request! I wanted to check in about the removal of the "link rel=" attributes. I believe we still want these "rel" tags. What is the benefit of removing them?

Reply:
I can double-check, but I believe these helped with making sure that Webpack loaded the styles properly. Once I removed these, the MIME type error no longer showed in the console.

Like I said, I'll double-check and get back to you before or during the standup. 👍